applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -1258,6 +1279,252 @@ static int cmd_program_info(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
+static int cmd_phy(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret;
+	uint8_t big_index;
+	uint8_t sub_index;
+
+	if (argc != 4) {
+		shell_error(shell,
+			    "Usage: bct phy <1, 2 or 4 (coded)> <BIG index> <subgroup index>");
+		return -EINVAL;
+	}
+
+	if (!is_number(argv[1])) {
+		shell_error(shell, "PHY must be a digit");
+		return -EINVAL;
+	}
+
+	uint8_t phy = (uint8_t)atoi(argv[1]);
+
+	if (phy != BT_AUDIO_CODEC_QOS_1M && phy != BT_AUDIO_CODEC_QOS_2M &&
+	    phy != BT_AUDIO_CODEC_QOS_CODED) {
+		shell_error(shell, "Invalid PHY");
+		return -EINVAL;
+	}
+
+	ret = argv_to_indexes(shell, argc, argv, &big_index, 2, &sub_index, 3);
+	if (ret) {
+		return ret;
+	}
+
+	broadcast_param[big_index].subgroups[sub_index].group_lc3_preset.qos.phy = phy;
+	broadcast_param[big_index].subgroups[sub_index].preset_name = "Custom";
+
+	return 0;
+}
+
+static int cmd_framing(const struct shell *shell, size_t argc, char **argv)
```

Comment:
Suggest we give these static functions a short doc, or expand a bit in the shell description at the bottom of this file.